lib/data_update_scripts/20220126102900_create_home_navigation_link.rb

```diff
@@ -0,0 +1,14 @@
+module DataUpdateScripts
+  class CreateHomeNavigationLink
+    def run
+      NavigationLink.where(url: "/").first_or_create(
```

Comment:
@citizen428 - I've had a crack at adding the update script for this - if you could take a look and let me know what needs amended that would be awesome 🙏

Reply:
This looks good @aitchiss! One thing we could potentially do is 

1. increment all positions of all existing navigation links by 2 and
2. insert this at position 1.

The only reason I'm mentioning this is to close the loop on the other discussion we had about this, I don't think it's worth it.